projects/packages/my-jetpack/_inc/components/onboarding-screen/styles.module.scss

```diff
@@ -1,7 +1,11 @@
 :global(body) {
 	font-family: "SF Pro Text", -apple-system, BlinkMacSystemFont, "Avenir Next", "Avenir", "Segoe UI", "Helvetica Neue", "Helvetica", "Cantarell", "Ubuntu", "Roboto", "Noto Sans", "Arial", sans-serif;
+}
+
+:global(#wpwrap) {
 	-webkit-font-smoothing: antialiased;
 	-moz-osx-font-smoothing: grayscale;
+	text-rendering: optimizeLegibility;
```

Comment:
Now I realized that we introduce these props to the global main WP element. Should we limit it just to our onboarding container to limit any styles pollution?

Reply:
We do, but the style is removed as soon as the component is unmounted.